### ISSUE TITLE
Create way to do identity proof on CMS/blog systems

### ISSUE THREAD

**Issue by brandelune** [keybase/client#11549]
It does not seem possible to get an identity proof from a blogspot blog.

**Comment by 5h4d0ww0lf** [comment]
Just claim it create message `I %username% hereby claim that I am %username% on keybase and as proof, I am signing this message` and sign it with saltpack.

**Comment by brandelune** [comment]
Sure, but I don't want that kind of message to appear on the blog itself.

**Comment by 5h4d0ww0lf** [comment]
You need public proof so it has to be on the blog itself publicly reachable.

**Comment by 5h4d0ww0lf** [comment]
Yeah and if you can't proof mac4translators.blogspot.jp by uploading a file to /.well_known/ then revoke it

**Comment by brandelune** [comment]
I understand that. But it should not have to be a page that it physically linked to the blog top page. Just like the proof for homepages.

**Comment by dabura667** [comment]
Be more specific.

**Comment by 5h4d0ww0lf** [comment]
@brandelune It has to be linked to the account or page. Webs are proven by placing a file in ./well_known/xxxxxxxx where is written what I wrote and this proves that you have access to the file system under webpage or through DNS record by placing txt record to your zone which proves that you have access to zone. If you want to prove blog where you don't have acces to anything other than blog itself what would you want other than public message on blog?
